# HG changeset patch
# User Kai T. Ohlhus <k.ohlhus@gmail.com>
# Date 1515594600 -3600
#      Wed Jan 10 15:30:00 2018 +0100
# Branch stable
# Node ID 06eb006b1e0fc46f9f99effc3908417007e71040
# Parent  6523facc5d8101537bd02f2a86fc000767ce5f77
doc: Correct fieldname of returned struct in ver (bug #52845).

* scripts/miscellaneous/ver.m: Revision ==> Release.

diff --git a/scripts/miscellaneous/ver.m b/scripts/miscellaneous/ver.m
--- a/scripts/miscellaneous/ver.m
+++ b/scripts/miscellaneous/ver.m
@@ -35,21 +35,21 @@
 ##
 ## @table @code
 ## @item Name
 ## Package name.
 ##
 ## @item Version
 ## Version of the package.
 ##
-## @item Revision
-## Revision of the package.
+## @item Release
+## Release of the package.
 ##
 ## @item Date
-## Date of the version/revision.
+## Date of the version/release.
 ## @end table
 ##
 ## @seealso{version, usejava, pkg}
 ## @end deftypefn
 
 ## Author: William Poetra Yoga Hadisoeseno <williampoetra@gmail.com>
 
 function retval = ver (package = "")
